app/components/newsletter/newsletter-header-overview.js

```diff
@@ -181,6 +190,40 @@ export default class NewsletterHeaderOverviewComponent extends Component {
     this.showConfirmPublishAll = false;
   }
 
+  async canSendMailCampaign() {
+    let themisPublicationActivities = await this.args.meeting.themisPublicationActivities;
+    let themisPublicationActivity = themisPublicationActivities.find((activity) => activity.scope.includes('documents'));
+
+    let hasDocumentPublicationPlanned = isPresent(themisPublicationActivity?.plannedDate);
+    let hasNotas = false;
+    let hasThemes = false;
+
+    const agendaitems = await this.store.query('agendaitem', {
+      filter: {
+        agenda: {
+          ':id:': this.args.agenda.id,
+        },
+        'type': {
+          ':uri:': CONSTANTS.AGENDA_ITEM_TYPES.NOTA,
+        },
+      },
+      'page[size]': PAGE_SIZE.AGENDAITEMS,
+    });
+    for (const agendaitem of agendaitems.toArray()) {
+      hasNotas = true;
```

Comment:
Can this be set outside the for-loop as `const hasNotas = agendaitems.length > 0`?